srtcore/core.cpp

```diff
@@ -7488,9 +7595,16 @@ bool CUDT::updateCC(ETransmissionEvent evt, const EventVariant arg)
                 // On updated SRTO_OHEADBW don't change input rate.
                 // This only influences the call to withOverhead().
             }
+            else if (m_PacingMode == SRT_PACING_INPUTBW || m_PacingMode == SRT_PACING_HYBRIDBW)
```

Comment:
If you use `m_bHybridPacing` as boolean, this condition could be simplified.

Actually in this form this is wrong. Sampled estimated bandwidth is required in ESTINPUTBW and HYBRIDBW modes, not in INPUTBW.